MyProject/cls/Table.cs

```diff
@@ -1,59 +1,110 @@
 ﻿using System;
+using System.Windows;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
 using System.Text;
+using System.Threading;
+using static cls.Tbl;
+using System.Timers;
 
 namespace cls
 {
     public class Tbl:ISaveRecord
     {
+        public delegate void OrderChange(string change);
+       
+        public event OrderChange OrderChanged;
+
         public int TblNumber { get; private set; }
         public int Guests { get; private set; }
 
-        private int orderNumber = SetOrderNum();
-
         public ObservableCollection<string> Order = new ObservableCollection<string>();
-
-
+        
         public bool OrderExists { get; private set; }
 
-        public string Employee { get; private set; }
+        internal long _orderId { get;  set; }
 
+        private System.Timers.Timer timeOpened;
 
         public Tbl(int tableNumber)
         {
             this.TblNumber = tableNumber;
-            this.Order = null;
-        }
+            _orderId++;
 
-        internal static int SetOrderNum()
-        {
-            int lastOrderNum = 1; //get last order number from storage
-            return lastOrderNum + 1;
+            timeOpened = new System.Timers.Timer(7200000);
```

Comment:
better to use System.Threaqding.Timer, read more here: https://docs.microsoft.com/uk-ua/dotnet/api/system.threading.timer?view=netframework-4.8